hw/bitstream/vivado/BUILD

```diff
@@ -70,20 +70,3 @@ filegroup(
     output_group = "otp_mmi",
     tags = ["manual"],
 )
-
-bitstream_splice(
-    name = "fpga_cw310_mask_rom",
-    src = ":fpga_cw310_test_rom",
-    data = "//sw/device/silicon_creator/mask_rom:mask_rom_fpga_cw310_scr_vmem",
-    meminfo = ":rom_mmi",
-    tags = ["manual"],
-)
```

Comment:
We need to keep this one: there needs to be a target guaranteed to generate a bitstream from vivado spliced with the mask_rom regardless whether `--define bitstream=...` (this will be used to create release artifacts).

Reply:
OK, bringing this back. I was attempting to avoid duplication. Is this a place where clarity for the reader is more important?